suites/nautilus/rgw/tier-2_rgw_test-bucket-lifecycle-rgw-multisite-primary-to-secondary.yaml

```diff
@@ -0,0 +1,122 @@
+# Below are the multi-site test scenarios run on the secondary and verified the sync/io on the primary
+# ceph-rgw1 is master/primary site
+# ceph-rgw2 is slave/secondary site
+
+tests:
+  - test:
+      name: pre-req
+      module: install_prereq.py
+      abort-on-fail: true
+      desc: install ceph pre requisites
+  - test:
+      name: ceph ansible
+      module: test_ansible.py
+      clusters:
+        ceph-rgw1:
+          config:
+            ansi_config:
+              ceph_test: True
+              ceph_origin: distro
+              ceph_repository: rhcs
+              osd_scenario: lvm
+              osd_auto_discovery: False
+              journal_size: 1024
+              ceph_stable: True
+              ceph_stable_rh_storage: True
+              fetch_directory: ~/fetch
+              copy_admin_key: true
+              dashboard_enabled: False
+              rgw_multisite: true
+              rgw_zone: US_EAST
+              rgw_zonegroup: US
+              rgw_realm: USA
+              rgw_zonemaster: true
+              rgw_zonesecondary: false
+              rgw_zonegroupmaster: true
+              rgw_zone_user: synchronization-user
+              rgw_zone_user_display_name: "Synchronization User"
+              rgw_multisite_proto: "http"
+              system_access_key: 86nBoQOGpQgKxh4BLMyq
+              system_secret_key: NTnkbmkMuzPjgwsBpJ6o
+              ceph_conf_overrides:
+                global:
+                  osd_pool_default_pg_num: 64
+                  osd_default_pool_size: 2
+                  osd_pool_default_pgp_num: 64
+                  mon_max_pg_per_osd: 1024
```

Comment:
Do we need these overrides? If yes, it would be nice to add some documentation around it in the top docstring

Reply:
I just had a discussion with Uday, we do not need these overrides. Have removed them.